### ISSUE TITLE
Things that would be nice to have in Tech Reborn without IC2 (or just in general), my thoughts

### ISSUE THREAD

**Issue by BBoldt** [TechReborn/TechReborn#316]
Thought I would just condense everything into a list here, these are just general thoughts that have come to mind while dealing with the 1.8.9 version of Tech Reborn sans IC2.

Quality of Life improvements:
* Hammer - new item, for making plates
* Rolling machine recipes for all plates
* Industrial grinder recipes for ALL ores, so that ore processing is actually possible, currently it only handles ores added by tech reborn
* There is no way to grind things into their dusts, maybe industrial grinder recipes for those as well
* Smelting recipes from all dusts to ingots. Currently the only ones active are the ones in the Industrial Blast furnace. (I can't smelt copper dust into copper ingots in a vanilla furnace)
* Display energy as RF when RF mode is on

Rambling opinion time:
I almost think it might be a good idea to port or rewrite the machines from the Advanced Machines mod in Tech Reborn, along with the induction furnace and the generator. That way you would have pretty much all of the base machines and parts without needing to have IC2 present. Without those, you're pretty much forced to go straight to the multiblocks, which don't seem to be fully working at the moment.

Along with that, you could have a few items, such as batteries and circuits. You already seem to have some circuit items, but they are uncraftable and don't do anything. It also might be useful to add a cable, even one would probably be enough.

This would also solve a lot of issues when it comes to recipes, as it would be easier to base recipes on those machines. Then it would just come down to changing the recipes for those depending on what mods are present with tech reborn.

**Comment by modmuss50** [comment]
Thanks again :)

I like the idea of a hammer(Can I have a texture please) and adding the recipes to the rolling machine. That is definitely possible.

I should be able to make the grinder work for all ores easily, im not sure why that is not a thing.
I can also add the dust recipes to this.

Smelting recipes should also be there not sure why they are not.

It will take a bit of work to display EU as RF but im sure I can do that.

I am looking into making a TechReborn port of advanced machines, It uses a lot of core ic2 things so I might be a bit of work, but definitely possible.

I have been holding off on the cables because I have been waiting on MCMultipart. But that is now out so I can get around to doing them.

**Comment by SwordKorn20** [comment]
You made a comment on CurseForge that you wanted this to be a standalone mod. So far, especially noticeable in 1.8.9, is the lack of early tier machinery akin to IC2.

I like the sound of this mod a lot but without a 1.8.9 release of IC on the horizon; this mod needs some work to make it truly standalone!

That being said, great work guys!

**Comment by modmuss50** [comment]
cables are in a separate mod: http://minecraft.curseforge.com/projects/fluxed-redstone
And  yes, a lot of things are missing to allow a fun gameplay experience in 1.8, atm its more of a prototype to show things are possible in 1.8

**Comment by SwordKorn20** [comment]
Oh I have fluxed redstone already! Didn't realise you guys had made them compatible!

**Comment by modmuss50** [comment]
Techreborn works with rf (Is a config setting - on by defualt in 1.8) even thought the guis are in EU

**Comment by modmuss50** [comment]
see #339
